clang/include/clang/CIR/Interfaces/CIRFPTypeInterface.td

```diff
@@ -0,0 +1,52 @@
+//===- CIRFPTypeInterface.td - CIR FP Interface Definitions -----*- C++ -*-===//
```

Comment:
Wasn't there some discussion on the last patch about the header not containing certain info?  I see this has the top level header, and the above file (CIRFPTypeInterface.h) having the description.  Which were we supposed to skip?

Reply:
You're right.  The updated guidelines say that this line doesn't need to contain any useful information.  I'll remove it.

Reply:
If you would, please audit the rest of the files that get added, I thought I noticed it a few places scrolling through.

Reply:
Done.  `CIRFPTypeInterface.cpp` had the same issue, which I fixed in a soon-to-be-commited change.

I don't plan to make this change in any existing files that were added in earlier commits, only in new files going forward.

Reply:
Yep, makes sense to me.